.github/workflows/ibis-backends.yml

```diff
@@ -96,9 +96,6 @@ jobs:
       - name: download backend data
         run: just download-data
 
-      - name: install backend data
-        run: poetry run python ci/datamgr.py load ${{ matrix.backend.name }}
-
       - name: run non-pyspark tests
         if: ${{ matrix.backend.name != 'pyspark' }}
         run: just ci-check -m ${{ matrix.backend.name }} --numprocesses auto --dist=loadgroup
```

Comment:
What is `just ci-check`?

Reply:
If you run `just` (it defaults to running `just --list`) inside the ibis repo you can see descriptions of each command:

```
❯ just
just --list
Available recipes:
    bench *args               # run the benchmark suite
    check *args               # run all non-backend tests; additional arguments are forwarded to pytest
    checklinks *args          # check for invalid links in a locally built version of the docs
    ci-check *args            # run pytest for ci; additional arguments are forwarded to pytest
    clean                     # clean untracked files
    default                   # list justfile recipes
    down                      # stop and remove containers; clean up networks and volumes
    download-data owner="ibis-project" repo="testing-data" rev="master" # download testing data
    fmt                       # format code
    lint                      # lint code
    list-backends             # show all backends
    load-data *backends="all" # load data into running backends; requires a running backend
    test +backends            # run the test suite for one or more backends
    typecheck                 # type check code using mypy
    up *backends              # start backends using docker compose and load data; no arguments starts all backends
```